### ISSUE TITLE
[Accessibility] Color Contrast is not respected.

### ISSUE THREAD

**Issue by andreacantone** [kebernet/shortyz#168]
The contrast ratio between text and background should be at least 4.5 . The following image shows the captions of the puzzles that do not respect the contrast ratio. 

![IMG_20201117_152248](https://user-images.githubusercontent.com/25429624/99405099-a6c12200-28ec-11eb-8d51-c49deab9579d.png)

> **How to reproduce the error**

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app [Shortyz Crosswords](https://play.google.com/store/apps/details?id=com.totsp.crossword.shortyz)
3. Activate the registration from the Accessibility Scanner and use the Shortyz Crosswords app.
4. After using the app, stop the recording.

The results will show the errors found in the app. For example:

![IMG_20201117_152224](https://user-images.githubusercontent.com/25429624/99405214-ce17ef00-28ec-11eb-87a0-54ad6f6a04da.png)
